src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -9708,6 +9714,7 @@ namespace OutputReportTabular {
 								windowAreaW += curArea * mult;
 								if ( isConditioned ) windowAreaWcond += curArea * mult;
 							}
+							zoneOpeningArea( zonePt ) += curArea * mult; // total window opening area for each zone (glass plus frame area)
```

Comment:
@JasonGlazer multiplier is applied here and below - looks like it gets multiplied twice?  Will make some test files to confirm.